rosa_tutorials/rosa-mobb-aws-cloudwatch-publish-metrics.adoc

```diff
@@ -0,0 +1,177 @@
+----
+$   export AWS_PAGER=""
+----
+
+. Set the environment variables:
++
+Change the values to suit your environment.
++
+[source,terminal]
+----
+$   export CLUSTER_NAME=metrics
+$   export CLUSTER_REGION=us-east-2
+$   export SCRATCH_DIR=/tmp/scratch
+$   mkdir -p $SCRATCH_DIR
+----
+
+. Create an AWS IAM user for CloudWatch:
++
+[source,terminal]
+----
+$   aws iam create-user \
+     --user-name $CLUSTER_NAME-cloud-watch \
+     > $SCRATCH_DIR/aws-user.json
+----
+
+. Fetch access and secret keys for the IAM user:
++
+[source,terminal]
+----
+$   aws iam create-access-key \
+     --user-name $CLUSTER_NAME-cloud-watch \
+     > $SCRATCH_DIR/aws-access-key.json
+----
+
+. Attach a policy to AWS IAM user:
++
+[source,terminal]
+----
+$   aws iam attach-user-policy \
+     --user-name $CLUSTER_NAME-cloud-watch \
+     --policy-arn "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
+----
+
+== Deploying the CloudWatch Prometheus agent
+
+. Create a namespace for CloudWatch:
++
+[source,terminal]
+----
+$   oc create namespace amazon-cloudwatch
+----
+
+. Download the CloudWatch agent Kubernetes manifests:
++
+[source,terminal]
+----
+$   wget -O $SCRATCH_DIR/cloud-watch.yaml https://mobb.ninja/docs/rosa/metrics-to-cloudwatch-agent/cloud-watch.yaml
+----
+
+. Update the CloudWatch agent Kubernetes manifests:
++
+[source,terminal]
+----
+$   sed -i .bak "s/__cluster_name__/$CLUSTER_NAME/g" $SCRATCH_DIR/cloud-watch.yaml
+$   sed -i .bak "s/__cluster_region__/$CLUSTER_REGION/g" $SCRATCH_DIR/cloud-watch.yaml
+----
+
+. Provide AWS credentials to the CloudWatch agent:
++
+[source,terminal]
+----
+$   AWS_ID=`cat $SCRATCH_DIR/aws-access-key.json | jq -r '.AccessKey.AccessKeyId'`
+$   AWS_KEY=`cat $SCRATCH_DIR/aws-access-key.json | jq -r '.AccessKey.SecretAccessKey'`
+
+$   echo "[AmazonCloudWatchAgent]\naws_access_key_id = $AWS_ID\naws_secret_access_key = $AWS_KEY" \
+     > $SCRATCH_DIR/credentials
+
+$   oc --namespace amazon-cloudwatch \
+     create secret generic aws-credentials \
+     --from-file=credentials=$SCRATCH_DIR/credentials
+----
+
+. Allow the CloudWatch agent to run as a root user inside the container:
++
+[source,terminal]
+----
+$   oc -n amazon-cloudwatch adm policy \
+     add-scc-to-user anyuid -z cwagent-prometheus
+----
+
+. Apply the CloudWatch agent Kubernetes manifests:
++
+[source,terminal]
+----
+$   oc apply -f $SCRATCH_DIR/cloud-watch.yaml
+----
+
+. Check that the pod is running:
++
+[source,terminal]
+----
+$   oc get pods -n amazon-cloudwatch
+----
++
+You should see:
++
+[source,terminal]
+----
+   NAME                                  READY   STATUS    RESTARTS   AGE
+   cwagent-prometheus-54cd498c9c-btmjm   1/1     Running   0          60m
+----
+
+== Creating a sample dashboard
+
+. Download the sample dashboard:
++
+[source,terminal]
+----
+$   wget -O $SCRATCH_DIR/dashboard.json https://raw.githubusercontent.com/rh-mobb/documentation/main/content/docs/rosa/metrics-to-cloudwatch-agent/dashboard.json
```

Comment:
The raw file cannot find with 404. Please make sure it can be reached.

Reply:
It should be https://raw.githubusercontent.com/rh-mobb/documentation/main/content/rosa/metrics-to-cloudwatch-agent/dashboard.json